documentation/carving/carving.md

```diff
@@ -10,7 +10,7 @@ weight: 4
 
 ## Carving Demo (4 minutes)
 
-<iframe width="560" height="315" src="https://www.youtube.com/embed/xGyTriPQXHI" frameborder="0" allowfullscreen></iframe>
+<iframe width="560" height="315" src="https://www.youtube.com/embed/xGyTriPQXHI?cc_load_policy=1" frameborder="0" allowfullscreen></iframe>
```

Comment:
Could also add `rel=0` query param to show related videos only from the ilastik YouTube channel.

Reply:
cool, I'll update all yt players!